src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/GerritCommentCountChangeNodeDecorator.java

```diff
@@ -58,16 +58,18 @@
     private ChangeInfo selectedChange;
     private Supplier<Map<String, List<CommentInfo>>> comments = setupCommentsSupplier();
     private Supplier<Map<String, List<CommentInfo>>> drafts = setupDraftsSupplier();
+    private Supplier<Set<String>> reviewed = setupReviewedSupplier();
 
     @Inject
     public GerritCommentCountChangeNodeDecorator(SelectedRevisions selectedRevisions) {
         this.selectedRevisions = selectedRevisions;
         this.selectedRevisions.addObserver(new Observer() {
             @Override
             public void update(Observable o, Object arg) {
-                if (arg != null && arg instanceof String && selectedChange.id.equals(arg)) {
+                if (arg instanceof String && selectedChange.id.equals(arg)) {
                         comments = setupCommentsSupplier();
                         drafts = setupDraftsSupplier();
+                        reviewed = setupReviewedSupplier();
```

Comment:
You also need to add this also in `onChangeSelected` (yes, because you have forgot to add it there, it shows some flaws in this code; you might could extract these common parts).